src/ui/components/Explorer.js

```diff
@@ -36,6 +55,9 @@ export type ExplorerState = {|
   showWeightConfig: boolean,
   view: CredView,
   recalculating: boolean,
+  anchorEl: any,
+  selectedIndex: number,
```

Comment:
Based on my testing, `selectedIndex` can be removed, since we aren't highlighting the selection in the menu itself.